proxy/src/read.rs

```diff
@@ -132,6 +132,15 @@ impl<Q: QueryExecutor + 'static> Proxy<Q> {
                 msg: format!("Failed to create plan, query:{sql}"),
             })?;
 
+        if let Some(table_name) = &table_name {
+            if !self.valid_ttl_range(&plan, catalog, schema, table_name) {
+                return Err(Error::SqlQueryOverTTL {
+                    code: StatusCode::OK,
+                    msg: format!("The query {sql}'s timestamp is already out of date"),
```

Comment:
`Time range of sql is over TTL, deadline:{}, sql:{}`

Reply:
fixed